### ISSUE TITLE
how to cite your result

### ISSUE THREAD

**Issue by ciwei123** [KaiyangZhou/deep-person-reid#131]
@KaiyangZhou I want to write my paper , and cite some of your conclusions.

KaiyangZhou.Pytorch implementation of deep person re-identification models[EB/OL].[2018-12].https://github.com/KaiyangZhou/deep-person-reid

Is this form correct?

**Comment by KaiyangZhou** [comment]
that's fine, you can simply put a footnote link

**Comment by ciwei123** [comment]
@KaiyangZhou Thank you very much!

**Comment by ciwei123** [comment]
@KaiyangZhou Hi, Did you use the shufflenet network to remove GlobalPool and FC ? I can not find the GlobalPool and FC in shufflenet.py .

**Comment by KaiyangZhou** [comment]
the global avg pool is implemented as `F.avg_pool2d` in `forward`. there is no additional fc

**Comment by ciwei123** [comment]
@KaiyangZhou  fc  is equivalent to "self.classifier = nn.Linear(cfg[num_groups][2], num_classes)" ? Right?

**Comment by ciwei123** [comment]
@KaiyangZhou  Your project  treats  the problem of re-identification as a classification problem？

**Comment by KaiyangZhou** [comment]
I mean:
````
fc -> self.fc = nn.Linear(in_dim, out_dim)
classifier -> self.classifier = nn.Linear(out_dim, num_classes)
````

Yes, in the case of `SoftmaxEngine` it is formulated as a classification problem

**Comment by ciwei123** [comment]
@KaiyangZhou
fc -> self.fc = nn.Linear(in_dim, out_dim)
classifier -> self.classifier = nn.Linear(out_dim, num_classes)
That means, result of fc is given to classifier？right?
but the  I can not understand that , there is no fc, how to complete the classifier?

**Comment by KaiyangZhou** [comment]
the final feature vectors will be connected to a classifier

they can be features global-avg-pooled from the final conv feature maps or additional fc layers

**Comment by ciwei123** [comment]
@KaiyangZhou Principle of ShuffleNet is conv feature →global-avg-pooled→fc→classifier→end.

and your code is conv feature →global-avg-pooled→classifier→end.

I thought I should remove A in the test section instead of during training.

**Comment by KaiyangZhou** [comment]
the FC in shufflenet denotes the classification layer (its output has 1000 units/classes)

after googlenet and NIN, it becomes a common practice to adopt the global avg pooling layer before an classification layer

**Comment by ciwei123** [comment]
@KaiyangZhou Thank you for your patience. That means the  "self.classifier"  in your code  is equivalent to the "FC" in shufflenet.

**Comment by KaiyangZhou** [comment]
yep, so FC generally means "fully connected layer", i.e. implemented with `nn.Linear` in the sense of pytorch

the classifier is also a FC, but in the code I rather call it "classifier" as it's intuitive
